include/exadg/incompressible_navier_stokes/time_integration/time_int_bdf_dual_splitting.cpp

```diff
@@ -622,6 +634,7 @@ TimeIntBDFDualSplitting<dim, Number>::projection_step()
 
     if(this->print_solver_info() and not(this->is_test))
     {
+      // No need to print the number of mass iterations here since we should be using L2
```

Comment:
"should be using L2" is unclear in my opinion. The question is: is it possible to run this part of the code (penalty terms applied!) in combination with HDIV? Whether we allow this parameter combination is to be commented in `Parameters` in my opinion. Given that you did not obtain exactly divergence-free results for the dual-splitting scheme, one might indeed want to experiment with this parameter combination in order to gain understanding.